### PR TITLE
Show changes across all named apps (even if they have the same repo) …

### PR DESCRIPTION
**Pull request by cpennington**
…in the release wiki page

You can see an example page here: https://openedx.atlassian.net/wiki/display/~cale/2017-03-23+Release

### CODE REVIEW

File: tubular/confluence_api.py [hunk 1]
```diff
@@ -152,9 +160,8 @@ def diff(base, new):
             They must also have the keys 'environment', 'deployment', 'play', and 'ami_id'.
     """
     diff_items = []
-    for delta in sorted(set(version_deltas(base, new)), key=lambda delta: delta.base or delta.new):
-        version = delta.new or delta.base
-        diff_items.append(E.LI(diff_link(version.repo, delta.base, delta.new)))
+    for delta in sorted(set(version_deltas(base, new)), key=lambda delta: (delta.app, delta.base or delta.new)):
```
Comment (feanil) [inline tubular/confluence_api.py]: NIT: this was hard to read.  Maybe a comment saying something like `# Add diff or commit links as a sorted list to the page.`

File: tubular/confluence_api.py [hunk 2]
```diff
@@ -127,26 +130,25 @@ def diff_link(delta):
...
+        href = format_commit_url(delta.new)
     else:
         summary = u"{}...{}".format(delta.base.sha, delta.new.sha)
         href = u"{}/compare/{}...{}".format(
             delta.new.repo, delta.base.sha, delta.new.sha
         )
 
-    return E.A(
-        u"{} ({}): {}".format(
+    return (
```
Comment (doctoryes) [inline tubular/confluence_api.py]: Nit: Is this pair of parens still needed?

**Review by doctoryes — APPROVED** [review]
If it works, merge it.

File: tubular/confluence_api.py (hunk 1 above)
Comment (macdiesel) [reply tubular/confluence_api.py]: I did have to go over this a few times myself.

Comment (cpennington) [reply tubular/confluence_api.py]: Yeah, it grew a bit incrementally. I'll tease it apart to make it more obvious.

File: tubular/confluence_api.py (hunk 2 above)
Comment (cpennington) [reply tubular/confluence_api.py]: This is returning a tuple (I suppose it could return a list, as well... That would probably be clearer).

Comment (doctoryes) [reply tubular/confluence_api.py]: Right - I see that now. I suppose a list would be clearer.

**Pull request merged**